barretenberg/cpp/src/barretenberg/relations/ultra_relation_consistency.test.cpp

```diff
@@ -255,42 +256,55 @@ TEST_F(UltraRelationConsistency, EllipticRelation)
         const auto& x_3 = input_elements.w_r_shift;
         const auto& y_3 = input_elements.w_o_shift;
 
+        // In the EllipticRelation, q_l is interpreted as q_sign and can be +1 or -1. Here we explicitly set it to -1
+        // (arbitrary, could also be +1) because the relation algebra makes use of the assumption that q_sign^2 = 1.
+        // This allows writing the simplified constraint algebra in this test in a more straightforward way.
+        input_elements.q_l = FF(-1);
         const auto& q_sign = input_elements.q_l;
         const auto& q_elliptic = input_elements.q_elliptic;
         const auto& q_is_double = input_elements.q_m;
 
         SumcheckArrayOfValuesOverSubrelations expected_values;
         // Compute x/y coordinate identities
-
         {
-            // Contribution (1) point addition, x-coordinate check
-            // q_elliptic * (x3 + x2 + x1)(x2 - x1)(x2 - x1) - y2^2 - y1^2 + 2(y2y1)*q_sign = 0
+            auto y_diff = (q_sign * y_2 - y_1);
```

Comment:
Here I'm just rewriting the simplified version of the constraints in a form that makes the derivation from the explicit formulas more clear. Previously they were basically just duplicated from the relation itself which isnt that useful for confirming that they're doing what we think.